src/mosaic-node-manager.cc

```diff
@@ -64,8 +64,14 @@ namespace ns3 {
         m_wifiChannelHelper.SetPropagationDelay("ns3::ConstantSpeedPropagationDelayModel");
         Ptr<YansWifiChannel> channel = m_wifiChannelHelper.Create();
         m_wifiPhyHelper.SetChannel(channel);
-        m_waveMacHelper = NqosWaveMacHelper::Default();
-        m_wifi80211pHelper = Wifi80211pHelper::Default();
+        // ns3::WifiPhy::ChannelWidth|ChannelNumber|Frequency are set via ns3_federate_config.xml
+        m_wifiMacHelper.SetType ("ns3::AdhocWifiMac", "QosSupported", BooleanValue (true));
+        m_wifiHelper.SetStandard (WIFI_STANDARD_80211p);
+        m_wifiHelper.SetRemoteStationManager ("ns3::ConstantRateWifiManager",
+                                "DataMode", StringValue ("OfdmRate6MbpsBW10MHz"),
+                                "ControlMode", StringValue ("OfdmRate6MbpsBW10MHz"),
+                                "NonUnicastMode", StringValue("OfdmRate6MbpsBW10MHz"));
```

Comment:
The code style is not 100% consistent here. (missing space between after StringValue in this line